model/src/main/java/org/cbioportal/model/GenesetAlteration.java

```diff
@@ -0,0 +1,40 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+
+public class GenesetAlteration implements Serializable {
```

Comment:
What would be stored in ` geneticEntityId`? Would this be the external id (so sometimes `Entrez_gene_id`, sometime geneset external id)? Or would it just be the internal `geneticEntityId` which is now present in both `gene` and `geneset` tables? 

As a side note: I thought you wanted to avoid the geneticEntityId as much as possible. Could you ellaborate a bit on the reason for having it in `model` now?